src/styles/blocks/header.scss

```diff
@@ -0,0 +1,70 @@
+@import "../styles/utils/variables";
+@import "../styles/utils/mixins.scss";
+@import "../styles/utils/reset.scss";
+
+.header {
+  font-weight: 500;
+  font-size: 12px;
+
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 0 50px;
+  background-color: $whiteColor;
+  box-shadow: 8px 2px 4px rgba(0, 0, 0, 0.05);
+
+  .logo {
+    &__image {
+      display: block;
+      @include square(40px);
+    }
+  }
+
+  .nav {
+
+    &__list {
+      display: flex;
+      list-style: none;
+    }
+
+    &__item {
+      margin-right: 20px;
+
+      &:last-child {
+        margin-right: 0;
+      }
+    }
+
+    &__link {
+      display: flex;
+      align-items: center;
+      height: $headerHeight;
+      line-height: 14px;
+      color: $blackColor;
+      text-transform: uppercase;
+      text-decoration: none;
+      transition: color 0.3s;
+
+      &.is-active {
+        color: $blueColor;
+        position: relative;
+
+        &::after {
+          position: absolute;
+          content: "";
```

Comment:
it's hard to understand code with three or even more nested selectors
Try to keep your code as simple as possible